sql/src/main/java/io/crate/analyze/AlterTableAnalyzer.java

```diff
@@ -69,8 +70,15 @@ AlterTableRenameAnalyzedStatement analyzeRename(AlterTableRename node, SessionCo
             throw new IllegalArgumentException("Target table name must not include a schema");
         }
 
-        RelationName relationName = RelationName.of(node.table(), sessionContext.defaultSchema());
-        DocTableInfo tableInfo = schemas.getTableInfo(relationName);
+        RelationName relationName;
+        if (node.blob()) {
+            relationName = new RelationName(BlobSchemaInfo.NAME, node.table().getName().getParts().get(0));
+            assert BlobSchemaInfo.NAME.equals(relationName.schema()) : "schema name must be 'blob'";
```

Comment:
I think the assertion could be removed. Shouldn't be necessary to assert that the constructor works correctly.